common/lib/container-definitions/api-report/container-definitions.api.md

```diff
@@ -448,7 +448,7 @@ export const IRuntimeFactory: keyof IProvideRuntimeFactory;
 
 // @public
 export interface IRuntimeFactory extends IProvideRuntimeFactory {
-    instantiateRuntime(context: IContainerContext): Promise<IRuntime>;
+    instantiateRuntime(context: IContainerContext, existing?: boolean): Promise<IRuntime>;
 }
```

Comment:
The `stateful` property has been removed as it's not actually necessary. The `existing` flag is optional so a new loader can load an old runtime (with no parameter in the function definition) and just pass the parameter which will end up being ignored. So considering we're not changing the function name, there are no back-compat worries here.